docs/api/js/types.md

```diff
@@ -1,7 +1,7 @@
 # Types
 
 All the types which are used in the SDK are referenced here:
-
+<!-- TODO: prepare quality documentation for SDK types with proper description and explanation -->
```

Comment:
not sure if the doc comments inside the code are not enough (if there are no comments there, then we should add those)

If you still think better documentation is needed, then, please create a task in linear for it